genie-web/src/main/java/com/netflix/genie/web/tasks/job/JobMonitoringCoordinator.java

```diff
@@ -116,10 +123,10 @@ public JobMonitoringCoordinator(
         this.registry = registry;
         this.jobsDir = jobsDir.getFile();
         this.jobsProperties = jobsProperties;
+        this.jobSubmitterService = jobSubmitterService;
 
         // Automatically track the number of jobs running on this node
-        this.registry.mapSize("genie.jobs.running.gauge", this.jobMonitors);
-        this.registry.mapSize("genie.jobs.scheduled.gauge", this.scheduledJobs);
+        this.registry.mapSize("genie.jobs.running.gauge", this.jobs);
         this.registry.methodValue("genie.jobs.active.gauge", this, "getNumActiveJobs");
```

Comment:
maybe it's better to tag them with the status so that we can break them up that way?